zipline/_protocol.pyx

```diff
@@ -437,35 +446,39 @@ cdef class BarData:
     @check_parameters(('assets',), (Asset,))
     def can_trade(self, assets):
         """
-        For the given asset or iterable of assets, returns true if all of the
+        For the given asset or iterable of assets, returns True if all of the
         following are true:
-        1) the asset is alive for the session of the current simulation time
-          (if current simulation time is not a market minute, we use the next
-          session)
-        2) (if we are in minute mode) the asset's exchange is open at the
-          current simulation time or at the simulation calendar's next market
-          minute
-        3) there is a known last price for the asset.
+
+        1. The asset is alive for the session of the current simulation time
+           (if current simulation time is not a market minute, we use the next
+           session)
+        2. The asset's exchange is open at the current simulation time or at
+           the simulation calendar's next market minute
+        3. There is a known last price for the asset.
```

Comment:
Super minor but this bullet has a period at the end but `1` and `2` don't, not sure what you prefer.